lib/google_places/rectangle.rb

```diff
@@ -0,0 +1,19 @@
+module GooglePlaces
+  class Rectangle
+    def initialize(lat1, lng1, lat2, lng2)
```

Comment:
Alright, if you can make sure to document (in a comment) sufficiently what these variables do and mean, it's fine with me :smile: